portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,102 @@
-#content {
+* { 
+    font-family: 'Didact Gothic';
+    color: whitesmoke;
+}
+
+body {
+background-image: linear-gradient(#434343,#000000);
+}
+
+.nav-items {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #333;
+}
+
+.nav-item {
+    float: left;
```

Comment:
nit: less two space for indentation

Reply:
Please resolve conflict before merging - it can be done either on github or locally.